### PR TITLE
[onert] Revisit DepthwiseConv2D nnfw_api test

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit renames test cases to be simple and adds test cases related to Bias and Multiplier.

ONE-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

File: tests/nnfw_api/src/GenModelTests/one_op_tests/DepthwiseConv2D.test.cc [hunk 1]
```diff
@@ -19,27 +19,27 @@
...
+  std::vector<float> weight_data{0.0f, 1.0f, 2.0f, 3.0f, 4.0f, 5.0f};
   uint32_t weight_buf = cgen.addBuffer(weight_data);
-  std::vector<float> bias_data{1, 2, 3, 4};
+  std::vector<float> bias_data{0.0f, 0.0f};
   uint32_t bias_buf = cgen.addBuffer(bias_data);
-  int in = cgen.addTensor({{1, 3, 2, 2}, circle::TensorType::TensorType_FLOAT32});
-  int weight = cgen.addTensor({{1, 2, 2, 4}, circle::TensorType::TensorType_FLOAT32, weight_buf});
-  int bias = cgen.addTensor({{4}, circle::TensorType::TensorType_FLOAT32, bias_buf});
-  int out = cgen.addTensor({{1, 2, 1, 4}, circle::TensorType::TensorType_FLOAT32});
-  cgen.addOperatorDepthwiseConv2D({{in, weight, bias}, {out}}, circle::Padding_VALID, 1, 1, 2,
```
Comment (ragmani) [inline tests/nnfw_api/src/GenModelTests/one_op_tests/DepthwiseConv2D.test.cc:30]: Why don't you keep this test with `Padding_VALID`?

Comment (jyoungyun) [reply tests/nnfw_api/src/GenModelTests/one_op_tests/DepthwiseConv2D.test.cc:30]: Good point, I will add `Padding_VALID` test case.

**Review by ragmani — DISMISSED** [review]
LGTM

File: tests/nnfw_api/src/GenModelTests/one_op_tests/DepthwiseConv2D.test.cc [hunk 2]
```diff
@@ -19,27 +19,51 @@
 TEST_F(GenModelTest, OneOp_DepthwiseConv2D)
 {
   CircleGen cgen;
-  std::vector<float> weight_data{1, 2, 3, 4, -9, 10, -11, 12, 5, 6, 7, 8, 13, -14, 15, -16};
+  std::vector<float> weight_data{1.0f, 2.0f, 3.0f, 4.0f, -9.0f, 10.0f,  -11.0f, 12.0f,
+                                 5.0f, 6.0f, 7.0f, 8.0f, 13.0f, -14.0f, 15.0f,  -16};
```
Comment (hseok-oh) [inline tests/nnfw_api/src/GenModelTests/one_op_tests/DepthwiseConv2D.test.cc:23]: Why weight shape is {1, 2, 2, 2} and data size is 16? 

```suggestion
  std::vector<float> weight_data{1.0f, 2.0f, 3.0f, 4.0f, -9.0f, 10.0f,  -11.0f, 12.0f};
```

**Comment by hseok-oh** [comment]
@jyoungyun Please fix format

**Review by hseok-oh — APPROVED** [review]

**Review by ragmani — APPROVED** [review]

**Pull request merged**